api/v1beta1/disruption_webhook.go

```diff
@@ -508,6 +508,11 @@ func safetyNetCountNotTooLarge(r *Disruption) (bool, string, error) {
 		}
 	}
 
+	if namespaceThreshold >= 1 && clusterThreshold >= 1 {
```

Comment:
```suggestion
	if namespaceThreshold >= MaxNamespaceThreshold && clusterThreshold >= MaxClusterThreshold {
```
nit: use constant rather magic numbers

Reply:
I knew this looked wrong, great idea